src/odemis/acq/test/stream_test.py

```diff
@@ -3837,11 +3851,23 @@ def test_cl_ss(self):
 
         cls.emtDwellTime.value = 10e-6 # s
         cls.pixelSize.value *= 10
+        cls.axis_band.value = 0
```

Comment:
Please don't hard code the values for the band. They are enumerated, and might change. It's better to just pick 2 "random" values, from the choices. Something like:
`b0, b1 = list(cls.axisFilter.choices)[:2]`